source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -117,13 +116,12 @@ namespace aspect
     EllipsoidalChunk<dim>::EllipsoidalChunkGeometry::push_forward_topography(const Point<3> &phi_theta_d_hat) const
     {
       AssertThrow (dim == 3,ExcMessage ("This can currently only be used in 3d."));
-
       const double d_hat = phi_theta_d_hat[2]; // long, lat, depth
       Point<dim-1> phi_theta;
       const double rad_to_degree = 180/numbers::PI;
       if (dim == 3)
         phi_theta = Point<dim-1>(phi_theta_d_hat[0] * rad_to_degree,phi_theta_d_hat[1] * rad_to_degree);
-      const double h = topography->value(phi_theta);
+      const double h = topography != NULL ? topography->value(phi_theta) : 0;
```

Comment:
This seems like an unrelated bug fix, right? That's ok, but it's often simpler to put these things into a separate pull request because two smaller PRs are usually reviewer quicker than one larger one :-)

Reply:
This is needed to make the new test work :)